### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by sgtphips** [sgtphips/codelab-semantic-html#1]
Hey @chrisvfritz, can you take a look at this? It's [hosted here](moaning-sister.surge.sh) and meets the following [project](http://lansingcodelab.com/lessons/semantic-html/1) criteria:

- [x] include a `header` element with the name of your blog and a tagline
- [x] include a `footer` element with copyright information and a design credit
- [x] include either a working `video` or `audio` element (you can use other sources, but `http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4` is a good video source and `http://www.w3schools.com/tags/horse.mp3` is a good audio source)
- [x] include at least 2 blog posts (of at least one sentence each)
- [x] return no errors (warnings are OK) when uploaded to the [W3C Markup Validation Service](http://validator.w3.org/#validate_by_upload)
- [x] include the `main`, `article`, `section`, `blockquote`, `figure`, and `figcaption` elements, in contexts where they make semantic sense

**Comment by chrisvfritz** [comment]
Very nice! The only thing it's missing is a `figcaption` element. 

And although it's outside the scope of this lesson, you've put in some extra work with these styles, so I'll push you a little further. I noticed that you're applying some normalizing styles to compensate for strange defaults in various browsers, like funky margin around the body. What I'd recommend instead, as a more thorough solution, would be to build your styles on top of a project like [normalize.css](https://necolas.github.io/normalize.css/). It's community-maintained, so includes fixes for lots of weird quirks that it would take a single person ages to discover and compensate for. And even Twitter Bootstrap builds their styles on top of it. Give it a try and let me know what you think. :smiley:

**Comment by sgtphips** [comment]
I added the figcaption tag! I also looked in to normalize css and got a good feel for what it actually does. I appreciate the advice. Wouldn't it be great if every tool you could need for any task could be easily available? One can dream!

**Comment by chrisvfritz** [comment]
Yeah, tell me about! :smiley: That's why I keep talking to other developers. :shipit:
